k8s/openebs-operator.yaml

```diff
@@ -649,6 +649,8 @@ spec:
               fieldPath: metadata.namespace
         - name: OPENEBS_IO_ENABLE_ANALYTICS
           value: "false"
```

Comment:
Also, can you add one more variable in the operator - localpv-provisioner and m-apiserver

```
- name: OPENEBS_IO_INSTALLER_TYPE
  value: "openebs-operator"
```

The value will be `openebs-operator` when used in the openebs-operator.yaml, and the value will be "charts-helm" -- when used in charts.

Reply:
Done.